util/utility.py

```diff
@@ -219,6 +219,11 @@ def retry_timeline(expected, request, token, initiative_id, field, num_required=
     count = 0
     res = request(initiative_id, token, page)
 
+    if res.status_code == 404:
+        res = request(initiative_id, token, page)
+        while res.status_code == 404:
+            res = request(initiative_id, token, page)
+
```

Comment:
@Benedetta-fabbri what do you think about introducing a sleep here to avoid requests close one another?

Reply:
yes, it makes sense